lib/sorbet-rails/gem_plugins/kaminari_plugin.rb

```diff
@@ -0,0 +1,21 @@
+# typed: true
+class KaminariPlugin < SorbetRails::ModelPlugins::Base
+  # Kaminari generates a dynamic `page` method on ActiveRecord relations.
+  # https://github.com/kaminari/kaminari/blob/c5186f5d9b7f23299d115408e62047447fd3189d/kaminari-activerecord/lib/kaminari/activerecord/active_record_model_extension.rb#L15
+  sig { params(root: Parlour::RbiGenerator::Namespace).void.implementation }
+  def generate(root)
+    return unless @model_class.include?(Kaminari::ActiveRecordModelExtension)
+
+    # Get the configured Kaminari page method name, or fall back to 'page' if necessary.
+    page_method = T.unsafe(Kaminari).config.page_method_name || 'page'
+
+    ar_querying_rbi = root.create_module(model_relation_shared_module_name)
```

Comment:
This is fine but a more sustainable approach is to add a model-scoped Kaminari-plugin. Is `page` the only method added by the gem?

Reply:
I think it's the only method, yeah.

Reply:
I see, this is fine then 👍

Reply:
It actually adds a few other methods by way of including `Kaminari::PageScopeMethods` and `Kaminari::ActiveRecordRelationMethods`. For example, this is valid Kaminari: `User.page(1).per(10)` but currently only the `page` call is recognised.

Reply:
What do you think about adding an `includes Kaminari::PageScopeMethods` to the model as part of this plugin?

Reply:
That sounds fine to me :)

Reply:
https://github.com/chanzuckerberg/sorbet-rails/pull/228